tapfreighter/chain_porter.go

```diff
@@ -1128,6 +1128,40 @@ func (p *ChainPorter) stateStep(currentPkg sendPackage) (*sendPackage, error) {
 				return nil, fmt.Errorf("unable to estimate "+
 					"fee: %w", err)
 			}
+
+			log.Infof("estimated fee rate for parcel:, %s",
+				feeRate.FeePerKVByte().String())
+		}
+
+		minRelayFee, err := p.cfg.Wallet.MinRelayFee(ctx)
+		if err != nil {
+			p.unlockInputs(ctx, &currentPkg)
+
+			return nil, fmt.Errorf("unable to obtain "+
+				"minimum relay fee: %w", err)
+		}
+
+		// If the fee rate is below the minimum relay fee, we'll
+		// bump it up.
+		if feeRate < minRelayFee {
+			switch {
+			// If a fee rate was manually assigned for this parcel,
+			// we err out, otherwise we silently bump the feerate.
+			case addrParcel.transferFeeRate != nil:
+				// This case should already be handled by the
+				// `checkFeeRateSanity` of `rpcserver.go`. We
+				// check here again to be safe.
+				p.unlockInputs(ctx, &currentPkg)
+
+				return nil, fmt.Errorf("feerate does not " +
```

Comment:
super-nit: should return the `minrelayfee` here so the user can reattempt above that feerate.